bundles/org.openhab.core.model.persistence/src/org/openhab/core/model/persistence/extensions/PersistenceExtensions.java

```diff
@@ -182,9 +182,9 @@ public static HistoricItem historicState(Item item, AbstractInstant timestamp, S
      *
      * @param item the item to check for state changes
      * @param timestamp the point in time to start the check
-     * @return <code>true</code> if item state had changed, <code>false</code> if it hasn't or if the default
-     *         persistence service does not refer to a {@link QueryablePersistenceService}, or <code>null</code> if the
-     *         default persistence service is not available
+     * @return <code>true</code> if item state has changed, <code>false</code> if it did not change or if the default
```

Comment:
Isn't "has changed or has not changed" the correct English?

Reply:
It looks/reads better using the same verb in both cases.

Reply:
Absolutely. "has changed" is preferred.